mcc/Dispatcher.c

```diff
@@ -159,6 +164,7 @@ static IPTR mNew(struct IClass *cl, Object *obj, struct opSet *msg)
             setFlag(data->flags, FLG_ActiveOnClick);
             setFlag(data->flags, FLG_PasteStyles);
             setFlag(data->flags, FLG_PasteColors);
+            setFlag(data->flags, FLG_HScroll);
```

Comment:
Ok, you are right, so keep that setFlag() here and we will see what effect this might have.